config/config.go

```diff
@@ -162,6 +170,7 @@ type Config struct {
 	IstioLabels      IstioLabels       `yaml:"istio_labels,omitempty"`
 	KubernetesConfig KubernetesConfig  `yaml:"kubernetes_config,omitempty"`
 	API              ApiConfig         `yaml:"api,omitempty"`
+	Auth             AuthConfig        `yaml:"auth,omitempty"`
```

Comment:
I think this should be named something more explicit and clear. "auth" makes it seem like it could be used generally (i.e. for Kubernetes as well as OpenShift). If I understand the what is going on here, this feature is ONLY for OpenShift. Perhaps we need a main "openshift" config group and put this "auth" under it.

Reply:
`Auth` is for all authentication configuration, such as Strategy, which is not openshift-only.